src/state/AppState.ts

```diff
@@ -61,8 +61,8 @@ export enum TileValue {
 export enum Winner {
   Cross = 0,
   Circle = 1,
-  Draw = 2,
-  None = 3,
+  None = 2,
```

Comment:
Why?

Reply:
"Then None is equal to Empty and Draw to Destroyed"